src/client/clouds.cpp

```diff
@@ -191,19 +190,23 @@ void Clouds::updateMesh()
 
 		v2f p0 = v2f(xi,zi)*cloud_size + world_center_of_drawing_in_noise_f;
 
-		video::S3DVertex v[4] = {
-			video::S3DVertex(0,0,0, 0,0,0, c_top, 0, 1),
-			video::S3DVertex(0,0,0, 0,0,0, c_top, 1, 1),
-			video::S3DVertex(0,0,0, 0,0,0, c_top, 1, 0),
-			video::S3DVertex(0,0,0, 0,0,0, c_top, 0, 0)
-		};
-
 		const f32 rx = cloud_size / 2.0f;
 		// if clouds are flat, the top layer should be at the given height
 		const f32 ry = is3D() ? m_params.thickness * BS : 0.0f;
 		const f32 rz = cloud_size / 2;
 
 		bool soft_clouds_enabled = g_settings->getBool("soft_clouds");
+		bool shaded_clouds_enabled = soft_clouds_enabled && g_settings->getBool("enable_dynamic_shadows") && g_settings->getBool("enable_3d_clouds");
```

Comment:
I do not see shaded_clouds_enabled being used anywhere.

Reply:
Ah yeah this is left over from another feature that didn’t make it into the PR… I am a little busy atm but I‘ll try and get this fixed soon.